packages/expo-barcode-scanner/src/BarCodeScanner.tsx

```diff
@@ -15,6 +15,8 @@ const { BarCodeType, Type } = ExpoBarCodeScannerModule;
 
 const EVENT_THROTTLE_MS = 500;
 
+let warnAboutDeprecation = false;
```

Comment:
Wouldn't `warnedAboutDeprecation` fit better? 😅 
```suggestion
let warnedAboutDeprecation = false;
```

Reply:
I was just about to change it 😅